libraries/SPI/SPI.cpp

```diff
@@ -403,16 +426,24 @@ void ArduinoSPI::configSpiSci(arduino::SPISettings const & settings)
   uint32_t smr  = R_SCI0_SMR_CM_Msk;
 
   /* Configure CPHA setting. */
-  spmr |= (uint32_t) clk_phase << 7;
+  uint32_t mask = (ONE_BIT << SCI_CKPH_POS);
+  spmr &= ~(uint32_t)mask; // reset bit 0
+  spmr |= ((uint32_t) clk_phase << SCI_CKPH_POS);
 
   /* Configure CPOL setting. */
-  spmr |= (uint32_t) clk_polarity << 6;
+  mask = (ONE_BIT << SCI_CKPOL_POS);
```

Comment:
I'm not really happy with the generic ONE_BIT, TWO__BITS defines. Maybe consider using the FSP layer provided constants, i.e.
```diff
-mask = (ONE_BIT << SCI_CKPOL_POS);
+mask = (SPI_CLK_POLARITY_HIGH << SCI_CKPOL_POS);
```
? This one's a matter of taste though.